pkgs/nix-tools/darwin-rebuild.sh

```diff
@@ -16,6 +16,14 @@ showSyntax() {
   exit 1
 }
 
+sudo() {
+  if command sudo --help | grep -- --preserve-env= >/dev/null; then
+    command sudo -H --preserve-env=PATH "$@"
+  else
+    command sudo -H --preserve-env "$@"
```

Comment:
```suggestion
    command sudo -H "$@"
```

If we can't preserve just `$PATH`, I reckon it should be fine to just preserve `$HOME`